File: 백준/python/2581.py
def find(num):
    if num == 0 or num == 1:
        return False
    for i in range(2,int(num**(1/2))+1):
        if num % i == 0:
            return False
        
    return True

start = int(input())
end = int(input())
result = []
for i in range(start, end+1):

    if find(i) == True:
        result.append(i)
if len(result) == 0:
    print("-1")
else:
    print(sum(result))
    
    print(result[0])

File: 백준/C/2581.c
#include <stdio.h>
#include <math.h>

int main()
{
	int start, end, len, min, sum, flag;
	sum = 0;
	len = 0;
	flag = 0;
	scanf("%d",&start);
	scanf("%d",&end);
	
	for (int i = start; i <= end; i++ )
	{
		if(find(i) != -1)
		{
			if (flag == 0)
			{
				min = i;
				flag = 1;
			}
			len +=1;
			sum+=i;
		}
	}
	if(len != 0)
	{
		printf("%d\n",sum);
		printf("%d",min);
	}
	else
	{
		printf("-1");
	}
	
}

int find(int num)
{
	if (num == 1 || num == 0)
	{
		return -1;
	}
	int det = sqrt(num);
	int i =2;
	while(i <= det)
	{
		if(num % i ==0)
		{
			return -1;
		}
		i++;
	}
	return num;
}
